### ISSUE TITLE
Term uri starts with the subsite directory name in WP Multisite

### ISSUE THREAD

**Issue by pshemek** [wp-graphql/wp-graphql#2710]
### Description

Unlike page or post queries, when you run term queries (category, tag) on the subsite (subdirectory) of Multisite network, the uri field includes subdirectory name.

Term uri should not contain subdirectory name.

### Steps to reproduce

1. Set up WP Multisite with subsites as subdirectories.
2. On the subsite, run the following query in GraphiQL:

```
query MultisiteUrisQuery {
  pages {
    nodes {
      uri
    }
  }
  categories {
    nodes {
      uri
    }
  }
  tags {
    nodes {
      uri
    }
  }
}
```

3. As a result, you'll get the uris in the following format:

```
/
/sample-page/
/sample-page/subpage/
/mysubsite/category/uncategorized/
/mysubsite/tag/some-tag/
```

The proper behaviour would be to not include the subsite directory name ("mysubsite") in term query results.

```
/category/uncategorized/
/tag/some-tag/
```

### WPGraphQL Version

1.13.7

### WordPress Version

6.1.1

### PHP Version

7.4.30

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have disabled ALL plugins except for WPGraphQL.

- [X] Yes
- [ ] My issue is with compatibility with a specific WordPress plugin, and I have listed all my installed plugins (and version info) above.

**Comment by jasonbahl** [comment]
@josephfusco looks like this PR aims to resolve this issue (https://github.com/wp-graphql/wp-graphql/pull/2341)

**Comment by ChrisWiegman** [comment]
This is closed by PR #2953
